# PersonsKeypoints.msg
# 多个人体的关键点数据列表

std_msgs/Header header
PersonKeypoints[] persons  # 所有人体的关键点列表

